gnd/src/main/java/com/google/android/gnd/model/form/Field.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model.form;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+/** A single field on a form for which the user may provide data. */
+@AutoValue
+public abstract class Field {
+  public enum Type {
+    TEXT,
+    MULTIPLE_CHOICE
+  }
+
+  @Nullable
+  public abstract String getId();
+
+  @Nullable
+  public abstract Type getType();
+
+  @Nullable
+  public abstract String getLabel();
+
+  public abstract boolean isRequired();
```

Comment:
I think it makes sense to define it at the field level because requiredness (?) is specific to fields. For sections headings it won't make sense, and for subforms we might have something different like cardinality.

Reply:
That makes sense, thanks!